site/en/docs/extensions/mv3/network-requests/index.md

```diff
@@ -143,6 +144,7 @@ chrome.runtime.sendMessage(
               encodeURIComponent(request.itemId)},
     response => parsePrice(response.text()));
```

Comment:
It looks like the newline between these two blocks is no longer rendering. Is there a way to fix that?